server/src/test/java/io/crate/integrationtests/PostgresITest.java

```diff
@@ -1160,7 +1169,23 @@ public void test_insert_array_in_simple_query_mode() throws Exception {
         } catch (BatchUpdateException e) {
             throw e.getNextException();
         }
+    }
+
+    @Test
+    public void test_original_query_appears_in_jobs_log() throws Exception {
+        var properties = new Properties();
+        properties.setProperty("user", "crate");
+        properties.setProperty("preferQueryMode", "simple");
+        try (Connection conn = DriverManager.getConnection(url(RW), properties)) {
+            var stmt = "SET GLOBAL TRANSIENT stats.enabled = true";
```

Comment:
Wouldn't be better to set a session setting like the datestyle or timezone to avoid resetting global settings? e.g.:
```
SET timezone =      'Europe/Berlin';
```

Reply:
Very true, thx 🤦